### PR TITLE
Improve the section about repository management

### PR DESCRIPTION
**Pull request by msehnout**
See individual commits.

### CODE REVIEW

File: osbuild-composer/src/user-guide/managing-repositories.md [hunk 1]
```diff
@@ -60,14 +60,14 @@ To set your own repositories, create this directory if it does not exist already
 $ sudo mkdir -p /etc/osbuild-composer/repositories
 ```
 
-Based on the system you are running (see `/etc/os-release` if you are not sure), determine the name of a new JSON file:
+Based on the system, you want to build an image for, determine the name of a new JSON file:
```
Comment (achilleas-k) [inline osbuild-composer/src/user-guide/managing-repositories.md:63]: Superfluous comma
```suggestion
Based on the system you want to build an image for, determine the name of a new JSON file:
```

File: osbuild-composer/src/user-guide/managing-repositories.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ Then, create the JSON file with the following structure (or copy the file for yo
 ```
 Specify only one value for the following attributes: `metalink`, `mirrorlist`, or `baseurl`. The remaining fields are optional.
 
-For example, assuming that the host OS is Fedora 33 running on x86_64, create `/etc/osbuild-composer/repositories/fedora-33.json` with this content:
+For example, a repository for building a Fedora 33 image running on x86_64, create `/etc/osbuild-composer/repositories/fedora-33.json` with this content:
```
Comment (achilleas-k) [inline osbuild-composer/src/user-guide/managing-repositories.md:89]: The sentence fragment between commas doesn't really work.  I think something like the following would   be correct:
```suggestion
For example, for building a Fedora 33 image running on x86_64, create `/etc/osbuild-composer/repositories/fedora-33.json` with this content:
```

**Review by achilleas-k — CHANGES_REQUESTED** [review]
Nice!  Thanks for pinging me on this.  I have a couple of suggestions.

**Comment by msehnout** [comment]
> Nice! Thanks for pinging me on this. I have a couple of suggestions.

Perfect, thanks. I've applied both of your suggestions :-)

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**